api/tests/acceptance/application/organization-invitation-controller_test.js

```diff
@@ -174,4 +174,112 @@ describe('Acceptance | Application | organization-invitation-controller', () =>
     });
   });
 
+  describe('GET /api/organization-invitations/{id}', () => {
+
+    let organizationId;
+    let options;
+
+    context('Success cases', () => {
+
+      beforeEach(async () => {
+        organizationId = databaseBuilder.factory.buildOrganization().id;
+
+        const code = 'ABCDEFGH01';
+
+        const organizationInvitationId = databaseBuilder.factory.buildOrganizationInvitation({
+          organizationId,
+          status: OrganizationInvitation.StatusType.PENDING,
+          code
+        }).id;
+
+        options = {
+          method: 'GET',
+          url: `/api/organization-invitations/${organizationInvitationId}?code=${code}`,
+        };
+
+        await databaseBuilder.commit();
+      });
+
+      afterEach(async () => {
+        await databaseBuilder.clean();
+      });
+
+      it('should return 200 HTTP status code', async () => {
+        // when
+        const response = await server.inject(options);
+
+        // then
+        expect(response.statusCode).to.equal(200);
+      });
+    });
+
+    context('Error cases', () => {
+
+      let organizationInvitationId;
+
+      beforeEach(async () => {
+        const code = 'ABCDEFGH01';
+
+        organizationId = databaseBuilder.factory.buildOrganization().id;
+        organizationInvitationId = databaseBuilder.factory.buildOrganizationInvitation({
+          organizationId,
+          code
+        }).id;
+
+        options = {
+          method: 'GET',
+          url: `/api/organization-invitations/${organizationInvitationId}?code=${code}`,
+        };
+
+        await databaseBuilder.commit();
+      });
+
+      afterEach(async () => {
+        await databaseBuilder.clean();
+      });
+
+      it('should respond with a 400 - missing parameters if organization-invitation is requested without code', async () => {
+        // given
+        options.url = `/api/organization-invitations/${organizationInvitationId}`;
+
+        // when
+        const response = await server.inject(options);
+
+        // then
+        expect(response.statusCode).to.equal(400);
+      });
+
+      it('should respond with a 404 if organization-invitation is not found with given id and code', async () => {
+        // given
+        options.url = `/api/organization-invitations/${organizationInvitationId}?code=999`;
+
+        // when
+        const response = await server.inject(options);
+
+        // then
+        expect(response.statusCode).to.equal(404);
+      });
+
+      it('should respond with a 421 if organization-invitation already exists', async () => {
```

Comment:
faut de typo : `is organization is accepted`

Reply:
done